packages/editor/css-properties/keywordDescriptionsByProperty.ts

```diff
@@ -0,0 +1,1472 @@
+    'self-start':
+      'Aligns the item to its own start side based on writing mode and direction.',
+    'self-end':
+      'Aligns the item to its own end side based on writing mode and direction.',
+    'flex-start': 'Aligns items to the start edge in flex layout.',
+    'flex-end': 'Aligns items to the end edge in flex layout.',
+  },
+  'align-self': {
+    normal: 'Uses the alignment behavior defined by the layout mode.',
+    stretch:
+      'Stretches items to fill the available space along the alignment axis.',
+    baseline:
+      "Aligns the item's baseline with the baseline of the alignment context.",
+    unsafe: 'Allows alignment even if it may cause overflow or clipping.',
+    safe: 'Prevents alignment that would cause overflow or clipping.',
+    center: 'Places items at the center along the alignment axis.',
+    start: 'Aligns items to the start edge of the alignment axis.',
+    end: 'Aligns items to the end edge of the alignment axis.',
+    'self-start':
+      'Aligns the item to its own start side based on writing mode and direction.',
+    'self-end':
+      'Aligns the item to its own end side based on writing mode and direction.',
+    'flex-start': 'Aligns items to the start edge in flex layout.',
+    'flex-end': 'Aligns items to the end edge in flex layout.',
+  },
+  'align-tracks': {
+    normal: 'Uses the alignment behavior defined by the layout mode.',
+    baseline:
+      "Aligns the item's baseline with the baseline of the alignment context.",
+    'space-between':
+      'Distributes items with equal space between them and no space at the ends.',
+    'space-around': 'Distributes items with equal space around each item.',
+    'space-evenly':
+      'Distributes items with equal space between and around items.',
+    stretch:
+      'Stretches items to fill the available space along the alignment axis.',
+    unsafe: 'Allows alignment even if it may cause overflow or clipping.',
+    safe: 'Prevents alignment that would cause overflow or clipping.',
+    center: 'Places items at the center along the alignment axis.',
+    start: 'Aligns items to the start edge of the alignment axis.',
+    end: 'Aligns items to the end edge of the alignment axis.',
+    'flex-start': 'Aligns items to the start edge in flex layout.',
+    'flex-end': 'Aligns items to the end edge in flex layout.',
+  },
+  'alignment-baseline': {
+    baseline: 'Aligns the baseline of the box with the baseline of the parent.',
+    alphabetic:
+      'Aligns based on the alphabetic baseline used in Latin scripts.',
+    ideographic:
+      'Aligns based on the ideographic baseline used in East Asian scripts.',
+    middle: 'Aligns at the mathematical center of the text.',
+    central: 'Aligns based on the central baseline of the parent.',
+    mathematical: 'Aligns based on the mathematical text baseline.',
+    'text-before-edge':
+      'Aligns the top of the text to the parent\u2019s text-before-edge baseline.',
```

Comment:
It might be nice to replace all of these?

```suggestion
      "Aligns the top of the text to the parent's text-before-edge baseline.",
```